media-libs/lcms/lcms-2.9.recipe

```diff
@@ -68,6 +61,25 @@ BUILD_PREREQUIRES="
 	cmd:make
 	"
 
+PROVIDES_tools="
+	lcms${secondaryArchSuffix}_tools = $portVersion
+	cmd:jpgicc$commandSuffix = 3.2
+	cmd:linkicc$commandSuffix = 2.2
+	cmd:psicc$commandSuffix = 2.1
+	cmd:tificc$commandSuffix = 6.2
+	cmd:transicc$commandSuffix = 4.3
```

Comment:
what are these version numbers?

Reply:
running the cmd's in Terminal with "-v" yields their version

Reply:
searching github they aren't used by any other recipe

Reply:
I don't really see the benefits of having these versions.

Reply:
I'll remove them then :)